### PR TITLE
Ka routes

### PR DESCRIPTION
**Pull request by kamos1**
## What's this PR do?

_Describe the problem or feature in addition to a link to the issues. Delete any content that doesn't apply._

- This PR is for moving the queries and routes of the server.js file. 

## Pre-Merge TODOs
 - [ ] Screenshots Included
    - ANY change to the UI of the app must have screenshots/gifs
    - Hint: recordit is really great for this
 - [ ] Tests Added
   - ANY change that can be tested must include testing
 - [ ] Updated Documentation
   - Update your README if any new dependencies are added
 - [ ] Add follow up issues?
 - [ ] 2 PR reviews completed

## Learning

_Describe the research stage._
_Links to blog posts, patterns, libraries or add-ons used to solve this problem._

## Where should the reviewer start?

_What should the reviewer look at. Are there any potential red flags or concerns you have? How will the reviewer see the code (can they run it locally, etc?)_

### CODE REVIEW

**Review by hilvitzs — COMMENTED** [review]
Everything looks good!

File: routes/queries.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+  User.findOne({username: user})
+    .then((user) => Outcome.findAll({user_id: user.id, outcome_types_id: typeCheck(type)}))
+    .then((outcomes) => response.status(200)
+      .send({text: `You have ${outcomes.length} ${type} `}))
+    .catch((error) => response.status(500).send(error))
+}
+
+const count = (request, response) => {
+  const user = request.query.user_name;
```
Comment (letakeane) [inline routes/queries.js:64]: It lives in the function so there's no worries about hoisting, but the duplication of the variable 'user' could potentially be confusing for someone inheriting your codebase (maybe?)

File: server.js [hunk 2]
```diff
@@ -32,56 +10,7 @@ app.set('port', (process.env.PORT || 3000));
...
-  const user = request.query.user_name;
-  const type = textCleaner(request.query.text);
-
-  User.findOne({username: user})
-    .then((user) => Outcome.findAll({user_id: user.id, outcome_types_id: typeCheck(type)}))
-    .then((outcomes) => response.status(200)
-      .send({text: `You have ${outcomes.length} ${type} `}))
-    .catch((error) => response.status(500).send(error))
-});
+app.use('/', routes)
```
Comment (letakeane) [inline server.js:13]: This is a hell of a refactoring :D

**Review by letakeane — COMMENTED** [review]
👍

**Pull request merged**